bindings/nodejs/examples/wallet/09-mint-native-token.ts

```diff
@@ -0,0 +1,85 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { MintNativeTokenParams } from '@iota/sdk';
+
+import { getUnlockedManager } from './account-manager';
+
+// The circulating supply of the native token. `100` hex encoded
+const CIRCULATING_SUPPLY = '0x64';
+// The maximum supply of the native token. `100` hex encoded
+const MAXIMUM_SUPPLY = '0x64';
+
+// In this example we will mint a native token.
+//
+// Make sure that `example.stronghold` and `example.walletdb` already exist by
+// running the `how_tos/accounts-and-addresses/create-wallet` example!
+//
+// Rename `.env.example` to `.env` first, then run
+// yarn run-example ./wallet/09-mint-native-token.ts
+async function run() {
+    try {
+        // Create the wallet
+        const manager = await getUnlockedManager();
+
+        // Get the account we generated with `01-create-wallet`
+        const account = await manager.getAccount(
+            `${process.env.ACCOUNT_ALIAS_1}`,
+        );
+
+        console.log('Sending alias output transaction...');
+
+        // First create an alias output, this needs to be done only once, because an alias can have many foundry outputs
+        let transaction = await account
+            .prepareCreateAliasOutput()
+            .then((prepared) => prepared.send());
```

Comment:
Do you prefer the `.then()` over this?
```suggestion
        let preparedTransaction = await account.prepareCreateAliasOutput();
        let transaction = await preparedTransaction.send();
```
Could even be shorter, but I think that's not that clear anymore :trollface: 
```suggestion
        let preparedTransaction = await (await account.prepareCreateAliasOutput()).send();
```

Reply:
yea i didnt like the double await, wasnt sure on the `.then` vs the 2 variables. Other people have an opinion?

Reply:
I like .then